internal/pkg/bootstrap/handlers/start_message/start_message.go

```diff
@@ -12,7 +12,7 @@
  * the License.
  *******************************************************************************/
 
-package handlers
+package start_message
```

Comment:
We should come up with a different name for this package. Underscores aren't considered good style in package names.

See [https://blog.golang.org/package-names](https://blog.golang.org/package-names)
```
Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. 
```
What about just `startup`?